sdk/storage/Azure.Storage.Files.Shares/src/autorest.md

```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/98b600498947073c18c2ac5eb7c3c658db5a1a59/specification/storage/data-plane/Microsoft.FileStorage/stable/2024-11-04/file.json
+    - https://raw.githubusercontent.com/seanmcc-msft/azure-rest-api-specs/a85d0c7a53c366f0ddf4f504ffdee16cbc23f69b/specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json
```

Comment:
TODO change this when https://github.com/Azure/azure-rest-api-specs/pull/30269 merges